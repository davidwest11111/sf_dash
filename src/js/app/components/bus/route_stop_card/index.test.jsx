import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import RouteStopCard from './index';


describe('outputs the expected tree when', () => {
  let wrapper;

  test('(default)', () => {
    wrapper = shallow((
      <RouteStopCard routeTag="14">
        something
      </RouteStopCard>
    ));
  });

  afterEach(() => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
