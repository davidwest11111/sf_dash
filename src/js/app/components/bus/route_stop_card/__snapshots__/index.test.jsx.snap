// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`outputs the expected tree when (default) 1`] = `
<Card
  className="mb-2"
  tag="div"
>
  <CardBody
    className="d-flex align-items-center justify-content-between"
    tag="div"
  >
    <div
      className="route-tag"
    >
      14
    </div>
    <div
      className="route-tag-details"
    >
      something
    </div>
  </CardBody>
</Card>
`;
